data/css/blog.scss

```diff
@@ -0,0 +1,111 @@
+@import "project-setup";
```

Comment:
I suspect we don't even need that file (at all) anymore, ill remove from here for now.